metrics/notebooks/Efficiency Cascades.ipynb

```diff
@@ -152,7 +199,15 @@
     }
    ],
    "source": [
-    "fig = plt.figure(figsize=(12, 5))\n",
+    "fig = plt.figure(figsize=(16, 6.5))\n",
+    "app_order = [\"OpenMP\", \"Kokkos\", \"CUDA\", \"OpenACC\", \"OpenCL\"]\n",
```

Comment:
Note to self: This is the approach to calling the script to ensure stable ordering of the applications so the application colours stay the same. The script is relatively unmodified, but invoked through the notebook with a list of applications and their orderings. By default, the script will order the applications with the most supported platforms first.